contracts/BSCValidatorSet.sol

```diff
@@ -90,14 +98,18 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
     uint256 enterMaintenanceHeight;     // the height from where the validator enters Maintenance
     bool isMaintaining;
 
+    // BEP-126 Fast Finality
+    bytes voteAddress;
```

Comment:
Please double check that `bytes` indeed occupy 1 slot here.

Reply:
Checked